src/features/content-explorer/content-explorer/ContentExplorer.scss

```diff
@@ -72,22 +72,50 @@
         }
     }
 
+    .bdl-ContentExplorer-subheader {
+        display: flex;
+        align-items: center;
+
+        .checkbox-container,
+        .toggle-container {
+            margin: 0;
+        }
+
+        .content-explorer-select-all-checkbox {
+            margin-right: 4.45 * $bdl-grid-unit;
+            margin-left: auto;
+        }
+    }
+
     .content-explorer-select-all-container {
-        display: block;
-        height: 8 * $bdl-grid-unit;
+        display: flex;
+        align-items: center;
+        width: 100%;
```

Comment:
i think replace `width: 100%` with `flex-grow: 1` might be more appropriate

with `width: 100%` means it should have the width of the parent but what we want is only the width of the remaining space in the parent